spring-cloud-stream-binder-kstream/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-stream-binder-kstream</artifactId>
+	<packaging>jar</packaging>
+	<name>spring-cloud-stream-binder-kstream</name>
+	<description>Kafka Streams Binder Implementation</description>
+
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-stream-binder-kafka-parent</artifactId>
+		<version>1.3.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<dependencies>
+		<dependency>
```

Comment:
Why all these dependencies are still here?
Have you had a chance to play with them somehow or at least answer why the concern hasn't been addressed?

I don't mind to address it in the separate PR, though, but some feedback would be great to have any way.

Thanks